test/test_cpp_api_parity.py

```diff
@@ -393,9 +397,10 @@ def generate_attr_equality_checks(module,
 
                 init_arg_spec = self._get_python_module_init_arg_spec(module.__class__.__name__)
                 # NOTE: `init_arg_spec.args[0]` is `self`, which is not counted as a constructor arg in the API parity test.
-                python_constructor_arg_names = [x for x in init_arg_spec.args[1:] if x != 'has_parity']
+                python_constructor_arg_names = [
+                    x for x in init_arg_spec.args[1:] if x not in module_metadata.python_ignored_constructor_args]
                 for name, attr in module.__dict__.items():
-                    if name not in TORCH_NN_MODULE_IGNORED_ATTRS:
+                    if name not in list(TORCH_NN_MODULE_IGNORED_ATTRS) + module_metadata.python_ignored_attrs:
```

Comment:
You seem to use this again. Maybe put it into a helper local function and call it `ignored_attrs()`